modules/apps/foundation/configuration-admin/configuration-admin-action/src/main/java/com/liferay/configuration/admin/action/ConfigurationModelActionException.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.configuration.admin.action;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Drew Brokke
+ */
+public class ConfigurationModelActionException extends PortalException {
+
+	public ConfigurationModelActionException(String msg) {
```

Comment:
Let's not create this constructor since it's too generic. Let's create one that contains more useful data (such as the properties) and creates a message itself.